wafer/schedule/templates/wafer.schedule/penta_schedule.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 {% load i18n %}
 <schedule>
+  <generator name="wafer" version="{{ wafer_version }}" />
+  <version>{# FIXME: We have no schedule versions #}</version>
```

Comment:
It's not clear to me from the frab documentation what version is supposed to be. Is this something we should update whenever the schedule is edited? Or something else entirely?

Reply:
> Is this something we should update whenever the schedule is edited

That's my guess.

I'm thinking of implementing something like that (probably in django cache), to allow us to have ETAGs and handle `If-Modified-Since` requests against the schedule.